packages/services/f-feature-management/test/data/simple-fmconfig.json

```diff
@@ -0,0 +1,45 @@
+{
```

Comment:
Maybe not in this PR, but I feel like this folder could use its own README.

Reply:
Ticketed